src/App.jsx

```diff
@@ -1,15 +1,21 @@
-import { useState } from "react";
+// import { useState } from "react";
+import { Route, Routes } from "react-router-dom";
 import { Logo } from "./components/Header";
 import Header from "./components/Header";
-import Body from "./components/Body";
+import Home from "./Pages/Home";
+import MovieDetails from "./components/MovieDetails";
 
 function App() {
   return (
     <div>
       <Header>
         <Logo />
       </Header>
-      <Body />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/movie/:id" element={<MovieDetails />} />
+        <Route path="/" element={<MovieDetails />} />
+      </Routes>
```

Comment:
You can't have two route definitions for path "/".
Also, is there a reason why you have 2 routes with the same element as `<MovieDetails>`

Reply:
I used the second to route the Back to List link as I used Link instead of useNavigate.